src/ring_theory/ideal/operations.lean

```diff
@@ -26,7 +26,11 @@ variables [comm_semiring R] [add_comm_monoid M] [module R M]
 open_locale pointwise
 
 instance has_scalar' : has_scalar (ideal R) (submodule R M) :=
-⟨λ I N, ⨆ r : I, (r : R) • N⟩
+⟨submodule.map₂ (linear_map.lsmul R M : _)⟩
```

Comment:
Oh yeah, it's commited partially. Why do you need the ` : _`?

Reply:
It's possible it's no longer needed; but an earlier version didn't tyopecheck without it

Reply:
Since CI has passed, maybe we can merge as is. I can remove the `: _` and add the two lemmas in subsequent PRs.

Reply:
I removed the underscore